services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/directory_watcher.py

```diff
@@ -82,7 +78,7 @@ async def wrapper(*args: Any, **kwargs: Any) -> Optional[Any]:
 
 
 async def push_directory_via_nodeports(directory_path: Path) -> None:
```

Comment:
minor: would not be more explicit to say where you push the directory? or just say push directory or so... the via is something that could change, and will probably be forgotten...

Reply:
ok, let me remove the `_via_nodeports`